lib/base/Version.js

```diff
@@ -31,8 +30,14 @@ Version.prototype.absoluteUrl = function (uri) {
  * @param  {string} uri uri to transform
  * @return {string} transformed url
  */
-Version.prototype.relativeUrl = function (uri) {
-  return _.trim(this._version, '/') + '/' + _.trim(uri, '/');
+Version.prototype.relativeUrl = function(uri) {
+  if (typeof this._version !== 'string') {
+    if (uri === undefined) {
+      return '';
+    }
+    return uri.trim('/');
+  }
+  return this._version.trim('/') + '/' + uri.trim('/');
```

Comment:
I'm assuming this is [string.trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) function. This function only trims whitespace and does not take in any args. So I don't think this will work.